tests/test_eval.py

```diff
@@ -1,6 +1,10 @@
 # -*- coding: utf-8 -*-
 import os
+
 import pytest
+
+import info  # noqa: F401
+
```

Comment:
These empty lines seem unnecessary if you ask me.

Reply:
I'm using [PEP8](https://www.python.org/dev/peps/pep-0008/#imports) import groups.

> Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.
> 
> Imports should be grouped in the following order:
> 
> Standard library imports.
> Related third party imports.
> Local application/library specific imports.
> You should put a blank line between each group of imports.

Reply:
Thanks for the reminder.